earthpy/spatial.py

```diff
@@ -209,7 +220,9 @@ def bytescale(data, cmin=None, cmax=None, high=255, low=0):
     Byte scaling means converting the input image to uint8 dtype and scaling
     the range to ``(low, high)`` (default 0-255).
     If the input image already has dtype uint8, no scaling is done.
-    This function is only available if Python Imaging Library (PIL) is installed.
+    This function is only available if Python Imaging Library (PIL) is
```

Comment:
Is this true? I can't see below how PIL is used.

Reply:
oh totally not true!

Reply:
fixed!